tidy3d/components/geometry/polyslab.py

```diff
@@ -431,6 +431,15 @@ def _normal_2dmaterial(self) -> Axis:
             raise ValidationError("'Medium2D' requires the 'PolySlab' bounds to be equal.")
         return self.axis
 
+    def _update_from_bounds(self, bounds: Tuple[float, float], axis: Axis) -> PolySlab:
+        """Set the bounds of a geometry in the axis direction."""
+        if axis != self.axis:
+            raise ValidationError(
```

Comment:
probably `ValueError` makes more sense. these `ValidtionErrors` are used mainly for when a `pydantic.validator` fails when initializing an object